### ISSUE TITLE
EAC process doesn't cleanly exit when application is closed

### ISSUE THREAD

**Issue by Lyle-Tafoya** [Guy1524/wine#8]
Currently it is necessary to force kill EAC and any other lingering .exe and wineserver processes whenever an EAC-enabled game is closed.

**Comment by Guy1524** [comment]
Yes, this is due to EasyAntiCheat.sys co-opting dllhost.exe and never closing it.  The best solution for this should be to mark dllhost.exe a system process.  However, I'll need to do it before LdrInitializeThunk, as after that point the process is in EAC's hands.
